packages/react-native/src/theme/types.ts

```diff
@@ -54,14 +54,17 @@ export type StrictTokens = ReactNativeTokens<'required'>;
 // Everything optional for custom themes
 export type Tokens = ReactNativeTokens<'optional'>;
 
+// omitted from custom themes until used in another component/publicly available
+type OmittedComponents = 'checkbox' | 'divider' | 'tabs';
```

Comment:
Good catch! Actually, looks like icon is not used either outside of IconButton and radio not used outside of RadioGroup.

Reply:
We use `IconButton` in the `PasswordField` right? And we use a `RadioGroup` in `VerifyUser``?